lib/cookie_consent_functions.php

```diff
@@ -72,10 +75,12 @@ public static function cookie_consent_backend() {
                 "deny"    => rex_escape($deny_content),
                 "allow"   => rex_escape($allow_content),
                 "link"    => rex_escape($link_content),
-                "href"    => rex_escape($externer_link) . '' . rex_escape($interner_link),
-				"target"  => rex_escape($link_target_type)
+                "href"    => rex_escape($externer_link) . '' . rex_escape($interner_link)
             ],
-            "type"     => $mode
+            "type"     => $mode,
+			"elements" => [
+				"messagelink" =>'<span id="cookieconsent:desc" class="cc-message">{{message}} <a aria-label="learn more about cookies" tabindex="0" class="cc-link" href="{{href}}" target="'.$link_target_type.'">{{link}}</a></span>'
```

Comment:
`target` sollte optional sein oder?

Reply:
target ist generell bei cookie consent mit _blank als Standard definiert, laut html Standard wäre dies aber _self. Ist die frage ob es somit optional sein sollte? Oder meinst du das generell bei keiner Angabe nach dem Update der Standard ausgeführt wird?

Reply:
Ein Standardwert wird in L44 gesetzt, oder meintest du etwas anderes?

Reply:
Wobei man bei nicht setzen der Option auf den Standardwert vom Cookie Consent JS zurückgreifen könnte.

Reply:
> target ist generell bei cookie consent mit _blank als Standard definiert

Ah ok … ich dachte das Attribute wäre jetzt zusätzlich dazu gekommen.

Reply:
Ah ok, dann passt das von meiner Seite soweit